src/main/java/nextstep/ladder/view/ResultView.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.ladder.view;
+
+import nextstep.ladder.domain.*;
+
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public final class ResultView {
+
+    private static final String FIRST_COLUMN = "    |";
+    private static final String NEW_LINE = "\n";
+    private static final String CONNECTED_ROW = "-----|";
+    private static final String UNCONNECTED_ROW = "     |";
+
+    private ResultView() {
+        throw new IllegalCallerException("잘못된 객체생성 입니다.");
+    }
+
+    public static void drawResultMessage() {
+        System.out.println("\n실행결과\n");
+    }
+
+    public static void drawParticipants(Participants participants) {
+        String drewUserNames = drawUserNames(participants);
+        System.out.println(drewUserNames);
+    }
+
+    public static void drawLadder(Ladder ladder) {
+        String drewLines = drawLines(ladder.getLineColumns());
+        System.out.println(drewLines);
+    }
+
+    private static String drawLines(LineColumns lineColumns) {
+        return lineColumns.getColumns()
+                .stream()
+                .map(column -> FIRST_COLUMN + drawRows(column))
+                .collect(Collectors.joining(NEW_LINE));
+    }
+
+    private static String drawUserNames(Participants participants) {
+        return participants.getUserNames()
+                .stream()
+                .map(UserName::getName)
+                .map(name -> String.format("%5s", name))
+                .collect(Collectors.joining(" "));
+    }
```

Comment:
```suggestion
    private static String drawUserNames(Participants participants) {
        List<String> names = participants.toNames();
        return names
                .stream()
                .map(name -> String.format("%5s", name))
                .collect(Collectors.joining(" "));
    }
```

`UserName::getName`의 의존성에 대해서는, `Participants`에게 위임해도 좋을것 같아요!
`ResultView.java`는 단지 출력하고자 하는 문자열 리스트가 궁금한게 아닐까요?

외부에서 내부 데이터를 세세하게 알게되면 의존성이 많이 꼬이게 되는것 같아요! 🤔

Reply:
넵 그렇네요..!!
수정하겠습니다 👍

Reply:
위의 테스트코드 수정하면서 해당 부분도 함께 수정되었네요 😃 
applied cid: b27299e7c5eedea37111e695e473a28cc42cee36